packages/schema/docs/build/schema.md

```diff
@@ -256,19 +252,15 @@ Key | Required | Type | Description
 #### Examples
 
 * ```
-  {
-    authorizeUrl: { require: 'some/path/to/file.js' },
-    getAccessToken: { require: 'some/path/to/file2.js' }
-  }
+  { authorizeUrl: { require: 'some/path/to/file.js' },
+    getAccessToken: { require: 'some/path/to/file2.js' } }
   ```
 * ```
-  {
-    authorizeUrl: { require: 'some/path/to/file.js' },
+  { authorizeUrl: { require: 'some/path/to/file.js' },
     getAccessToken: { require: 'some/path/to/file2.js' },
     refreshAccessToken: { require: 'some/path/to/file3.js' },
     scope: 'read/write',
-    autoRefresh: true
-  }
+    autoRefresh: true }
```

Comment:
this big formatting change is usually the result of an out-of-date `prettier` installation.

Reply:
yikes, yeah I didn't realize the pre-commit would prettier that automatically! I'll make sure that's updated, thanks

Reply:
possibly silly question but -- where would the relevant prettier installation be, or alternately how do the schema docs get rebuilt? I updated prettier locally and `prettier -v` now returns the most recent version, but the pre-commit doesn't seem to be making further changes to the schema 🤔

Reply:
it would be in `node_modules` at the repo root. it's honestly not a big deal if this gets reformatted in weird ways, but I figured it was worth mentioning

Reply:
figured it out! I was using an old node version from working on an older app. should be all good now.